deployd/pkg/deployd/deployd.go

```diff
@@ -159,6 +159,8 @@ func Run(logger *log.Entry, req *deployment.DeploymentRequest, cfg config.Config
 
 		logger.Infof("Resource %d: successfully deployed %s", index+1, deployed.GetSelfLink())
 
+		logger := logger
+		resource := resource
```

Comment:
Consider shadowing these variables in the goroutine itself, such as:

```
go func(logger T, resource T)() {
...
}(logger, resource)
```

Reply:
That was my first thought, however I wasn't sure that it'd work. Fixing.